src/services/accounts/AccountsBalanceInfoService.spec.ts

```diff
@@ -18,5 +23,89 @@ describe('AccountsBalanceInfoService', () => {
 				)
 			).toStrictEqual(accountsBalanceInfo789629);
 		});
+
+		describe('token recognization', () => {
+			let tempQueryTokens: any,
+				tempQueryBalance: any,
+				mockTokensLocksAt: jest.Mock<any>,
+				mockTokenAccountAt: jest.Mock<any>,
+				mockBalancesLocksAt: jest.Mock<any>;
```

Comment:
Is that standard for the linter to indent multiple line `lets` like that?

Reply:
Yes and the 4 space tab looks pretty exagerated on github vs vscode. One of these days we can change all our repos to 2 space tabs to line up closer with the standard 2 spaces that most JS projects outside fo parity use.z